src/main/java/net/dv8tion/jda/core/entities/impl/ReceivedMessage.java

```diff
@@ -651,10 +651,12 @@ public Guild getGuild()
                 {
                     final long emoteId = MiscUtil.parseSnowflake(matcher.group(2));
                     final String emoteName = matcher.group(1);
+                    // Check animated by verifying whether or not it starts with <a: or <:
+                    final boolean animated = matcher.group(0).startsWith("<a:");
```

Comment:
group(0) is equivalent to just group(). The pattern doesn't seem to group that part so you probably don't mean 1.

Reply:
I did mean group 0, however I don't see a real reason to change this other than for the sake of changing it.